src/Tests/BusinessLayer/SerialReportingTest.cpp

```diff
@@ -631,6 +631,7 @@ TEST_F(SerialReportingTest, sendLightsTest) // TODO create function which build
     const unsigned int payloadLength = expectedPackageLength - COBS_ADDITIONAL_FRAME_DATA_SIZE;
     unsigned char data[payloadLength];
     data[0] = CcsDefines::LIGHTS_PKG_ID;
+    data[1] = CcsDefines::LIGHTS_ALIVE;
```

Comment:
There's no `LIGHTS_ALIVE` in `CcsDefines.h`. Lights Alive would be a boolean value. Check out how the battery_alive is represented in `BatteryData.h` and `SendBattery()`.